DayrSuryan/DSEthiop9.xml

```diff
@@ -58,9 +58,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <textLang mainLang="gez"/>
 
                      <incipit xml:lang="gez" type="inscriptio">
-                         <hi rend="rubric">በስመ፡ አብ ወወልድ፡ ወመንፈ<sic resp="DR">ን</sic>ስ፡ ቅዱስ፡</hi>
+                         <hi rend="rubric">በስመ፡ አብ ወወልድ፡ <sic resp="DR">ወመንፈንስ፡</sic> ቅዱስ፡</hi>
```

Comment:
I think that this change has been carried out to change the visualisation. @PietroLiuzzo the fact that `<sic>` is not yet rendered as (!) after the word (not the letter) does not mean that it is possible, right?

Reply:
@PietroLiuzzo could you confirm (or correct me)?